src/main/java/org/openrewrite/java/migrate/util/UseEnumSetOf.java

```diff
@@ -66,9 +65,7 @@ public J.MethodInvocation visitMethodInvocation(J.MethodInvocation method, Execu
                         if (isAssignmentSetOfEnum(type)) {
                             maybeAddImport("java.util.EnumSet");
 
-                            StringJoiner setOf = new StringJoiner(", ", "EnumSet.of(", ")");
                             List<Expression> args = m.getArguments();
-                            args.forEach(o -> setOf.add("#{any()}"));
```

Comment:
White the variable is indeed unused here, it seems like this currently doesn't work quite right when there are multiple parameters passed to the `Set#of()` method. It looks like a fix was this was started with this `args` variable. By using this `args.toString()` as the template, the recipe should also work for multiple parameters.

Reply:
Ah ok, I've updated the test to use multiple parameters, and adapted the recipe